src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticSourceEventSource.cs

```diff
@@ -33,18 +33,18 @@ namespace System.Diagnostics
     ///   
     ///   * It is a newline separated list of FILTER_AND_PAYLOAD_SPEC
     ///   * a FILTER_AND_PAYLOAD_SPEC can be 
-    ///       * EVENT_NAME : TRANSFORM_SPEC
+    ///       * EVENT_NAME : TRANSFORM_SPECS
     ///       * EMPTY - turns on all sources with implicit payload elements. 
     ///   * an EVENTNAME can be  
     ///       * DIAGNOSTIC_SOURCE_NAME / DIAGNOSTC_EVENT_NAME @ EVENT_SOURCE_EVENTNAME  - give the name as well as the EventSource event to log it under.  
     ///       * DIAGNOSTIC_SOURCE_NAME / DIAGNOSTC_EVENT_NAME   
     ///       * DIAGNOSTIC_SOURCE_NAME    - which wildcards every event in the Diagnostic source or 
     ///       * EMPTY                     - which turns on all sources
-    ///   * a TRANSFORM_SPEC can be 
-    ///       * - TRANSFORM_SPEC - the '-' indicates that implicit payload elements should be suppressed 
+    ///   * TRANSFORM_SPEC is a semicolon separted list of TRANSFORM_SPEC, which can be 
```

Comment:
No it is not recursive.  As you point out it is technically wrong, but the goal here is not a formal grammar but something that quickly tells humans how to use it.     Introducing either optional syntax or another grammar node probably make things harder to read.